setup.py

```diff
@@ -49,6 +49,7 @@
         "plone.resource",
         "plone.autoform",
         "plone.behavior",
+        "plone.restapi",
```

Comment:
plone.app.discussion should not have a hard dependency on plone.restapi.

Put the code which imports from plone.restapi in a separate module, then add `<configure zcml:condition="installed plone.restapi">` around the ZCML

Reply:
> plone.app.discussion should not have a hard dependency on plone.restapi.

ah, that slipped away frm my mind...

Reply:
@davisagli, I've added the changes however wasn't sure if naming is correct or not... please suggest if needed!